packages/cli/snapshots/integration/generators/relation.has-one.integration.snapshots.js

```diff
@@ -0,0 +1,421 @@
+// IMPORTANT
+// This snapshot file is auto-generated, but designed for humans.
+// It should be checked into source control and tracked carefully.
+// Re-generate by setting UPDATE_SNAPSHOTS=1 and running tests.
+// Make sure to inspect the changes in the snapshots below.
+// Do not ignore changes!
+
+'use strict';
+
+exports[`lb4 relation checks generated source class repository  answers {"relationType":"hasOne","sourceModel":"Order","destinationModel":"Customer"} generates Order repository file with different inputs 1`] = `
+import {DefaultCrudRepository, repository, HasOneRepositoryFactory} from '@loopback/repository';
+import {Order, Customer} from '../models';
+import {DbDataSource} from '../datasources';
+import {inject, Getter} from '@loopback/core';
+import {CustomerRepository} from './customer.repository';
+
+export class OrderRepository extends DefaultCrudRepository<
+  Order,
+  typeof Order.prototype.id
+> {
+
+  public readonly customer: HasOneRepositoryFactory<Customer, typeof Order.prototype.id>;
+
+  constructor(@inject('datasources.db') dataSource: DbDataSource, @repository.getter('CustomerRepository') protected customerRepositoryGetter: Getter<CustomerRepository>,) {
+    super(Order, dataSource);
+    this.customer = this.createHasOneRepositoryFactoryFor('customerError: bad input', customerRepositoryGetter,);
```

Comment:
I think some snapshots are incorrect ( i.e `'customerError: bad input'` part).

Let's fix the generator and template and fix the tests later